Examples/Install-Win32OpenSSH/Install-Win32OpenSSH.ps1

```diff
@@ -0,0 +1,173 @@
+#
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this
+# software and associated documentation files (the "Software"), to deal in the Software
+# without restriction, including without limitation the rights to use, copy, modify,
+# merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+# INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+# PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+# HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+# OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+
+$ErrorActionPreference = 'stop'
+$ProgressPreference = 'silentlycontinue'
+$VerbosePreference = 'continue'
+ 
+$WorkDir = 'c:\Amazon'
+if (!(Test-Path -Path $WorkDir)) {
+    mkdir -Path $WorkDir
+}
+Set-Location -Path c:\Amazon
+ 
+function Write-Log {
+  [CmdletBinding()]
+  param (
+    [Parameter(Mandatory = $true)]
+    [string] $Message
+  )
+  Add-Content -Path $env:SystemDrive\Win32-OpenSSH.log -Value ('{0}: {1}' -f (Get-Date -Format o), $Message)
+}
+ 
+function Install-PowerShellCore {
+    Write-Log -Message 'Installing PowerShell Core'
+    $SourceURL = 'https://github.com/PowerShell/PowerShell/releases/download/v6.0.0/PowerShell-6.0.0-win-x64.zip'
```

Comment:
Is there a reason for 6.0.0 instead of 6.0.1? 
https://github.com/PowerShell/PowerShell/releases/download/v6.0.1/PowerShell-6.0.1-win-x64.zip

Reply:
@markekraus Good call, I should update the PowerShell Core revision.